client/keys/add.go

```diff
@@ -90,7 +92,7 @@ input
 output
 	- armor encrypted private key (saved to file)
 */
-func runAddCmd(_ *cobra.Command, args []string) error {
+func runAddCmd(home string, args []string) error {
```

Comment:
I would prefer to keep cmd here.

Reply:
I agree with @jleni, can't we just add `home` to the `addKeyCommand` args and then define `runAddCmd` in the struct? That would fit with style elsewhere.